ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml

```diff
@@ -174,16 +210,17 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_TYPE: m6g.xlarge
       CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
-      E2E_RUN_TAGS: '@aws-ipi and @arm64 and @disconnected and @network-ovnkubernetes
-        and not @fips'
+      E2E_RUN_TAGS: '@arm64 and @aws-ipi and @disconnected and @network-ovnkubernetes
+        and @singlenode and not @fips and not @console'
       OCP_ARCH: arm64
       TAG_VERSION: '@4.12'
       TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&
+      TEST_SCENARIOS: MCO|ETCD|PSAP|SDN|API_Server|Authentication|Cluster_Operator|Cluster_Infrastructure|OLM|Network_Edge|Operator_SDK|Workloads|Image_Registry|Container_Engine_Tools|NODE|OTA|PerfScale|Cluster_Observability|Security_and_Compliance|LOGGING
```

Comment:
@aleskandro you do not take default value to remove STORAGE subteam. just double confirm with you: why remove STORAGE? because of this SNO?
if it is caused by SNO, test `aws-ipi-disconnected-private-sno-p3-f28-destructive` is sno, but I do not find STORAGE is removed.

this comment apply to other tests of this PR and also apply to other PRs in https://coreos.slack.com/archives/CPJQCSN49/p1671141487692949

Reply:
This is because `baselinecaps-none`, which does not install storage operator.
@duanwei33 has more context/detail.